src/main/scala/uk/gov/hmrc/ct/ct600e/v2/calculations/LoansAndDebtorsCalculator.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.ct600e.v2.calculations
+
+import uk.gov.hmrc.ct.ct600e.v2.{E24e, E24e2, E24e1}
+
+trait LoansAndDebtorsCalculator {
+  def calculateFieldValue(e24e1: E24e1, e24e2: E24e2): E24e = {
+    val fields = Set(e24e1, e24e2)
```

Comment:
This is the same as the E170 calc, we could move this out into a common calculation trait